### PR TITLE
terraform config for aws

### PR DESCRIPTION
**Pull request by oavdeev**
**What this PR does / why we need it**:
This contains an example tf config for setting up Feast on AWS. This includes: 
* VPC and subnets
* EKS cluster
* Redis (via Elasticache)
* Configuring and applying Helm charts via terraform helm provider
* Postgres DB (via serverless Aurora)
* Optional persistent EMR Spark cluster
* S3 bucket to store logs jars and temporary data

Does not (yet) include:
* Kafka

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: infra/terraform/aws/eks.tf [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  length  = 8
+  special = false
+}
+
+resource "aws_security_group" "all_worker_mgmt" {
+  name_prefix = "${var.name_prefix}-worker"
+  vpc_id      = module.vpc.vpc_id
+}
+
+module "vpc" {
```
Comment (jmelinav) [inline infra/terraform/aws/eks.tf:59]: Will it be possible to provide a way to use existing vpc?

File: infra/terraform/aws/eks.tf [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+  tags = {
+    Environment = "test"
+    GithubRepo  = "terraform-aws-eks"
+    GithubOrg   = "terraform-aws-modules"
+  }
+
+  vpc_id = module.vpc.vpc_id
+
+  worker_groups = [
```
Comment (jmelinav) [inline infra/terraform/aws/eks.tf:99]: May be this should be a variable with default values?

**Comment by woop** [comment]
We're going to merge this as an example. We can refactor the implementation into a module in a new PR. @jmelinav

**Review by woop — APPROVED** [review]

**Pull request merged**